csharp/src/Vista.SDK/LocalIdBuilder.cs

```diff
@@ -122,6 +122,27 @@ public LocalId Build()
         return new LocalId(this);
     }
 
+    public LocalIdErrorBuilder Validate(out LocalIdErrorBuilder errorBuilder)
```

Comment:
Since we now have `Validate` _and_ `IsValid` - do they check the same things?

Reply:
Validate uses both `IsValid `and `IsEmpty`